src/SDDPoptimize.jl

```diff
@@ -169,14 +164,14 @@ Return:
 Float64 (estimation of the upper bound)
 
 """
-function estimate_upper_bound(model, param, V, problems, n_simulation=1000)
+function estimate_upper_bound(model::SPModel, param::SDDPparameters, V::Vector{PolyhedralFunction}, problem::Vector{JuMP.Model}, n_simulation=1000::Int)
```

Comment:
Could this function work for both SDP and SDDP ? If forward_simulations has the same structure it chould work.